storybookjs/storybook
# CLI: Scope styles in sample components from the CLI templates

Closes #19409 , Closes #12423

## What I did

Redo of #19433, thank you so so much @Amerlander !

Added storybook specific classes to Page and Header components to prevent its CSS being applied to other elements.

This solution was discussed here before: https://github.com/storybookjs/storybook/pull/19410#issuecomment-1273626505

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Open Storybook in your browser
3. Access the basic CLI stories, they should not have changed (but their styles would not leak anymore!)

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
